packages/@tinacms/cli/src/cmds/audit/audit.ts

```diff
@@ -22,94 +22,39 @@ type AuditArgs = {
   collection: TinaCloudCollection
   database: Database
   rootPath: string
+  documents: { node: { path: string } }[]
   useDefaultValues: boolean
+  verbose?: boolean
 }
 export const auditCollection = async (args: AuditArgs) => {
   let warning = false
-  const { collection, database, rootPath } = args
+  const { collection } = args
   logger.info(`Checking collection ${collection.name}`)
-  const query = `query {
-        collection(collection: "${collection.name}") {
-          format
-          documents {
-            edges {
-              node {
-                ...on Document {
-                  _sys {
-                    extension
-                    path
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
-      `
-  const result = await resolve({
-    database,
-    query,
-    variables: {},
-  })
-  const format = result.data.collection.format
-  const docs = result.data.collection.documents.edges
 
-  // validate file extension
-  docs.forEach((x) => {
-    const node = x.node
-    if (node._sys.extension.replace('.', '') !== format) {
+  const format = collection.format
+  args.documents.forEach((x) => {
+    const docFormat = path.extname(x.node.path)
+    if (docFormat.replace('.', '') !== format) {
       warning = true
       logger.warn(
         chalk.yellowBright(
-          `WARNING: there is a file with extension \`${
-            node._sys.extension
-          }\` but in your schema it is defined to be \`.${format}\`\n\n\location: ${p.join(
-            rootPath,
-            node._sys.path
-          )}`
+          `WARNING: there is a file with extension \`${docFormat}\` but in your schema it is defined to be \`.${format}\`\n\n\location: ${x.node.path}`
         )
       )
     }
   })
-
   return warning
 }
 
 export const auditDocuments = async (args: AuditArgs) => {
-  const { collection, database, rootPath, useDefaultValues } = args
-  const query = `query {
-        collection(collection: "${collection.name}") {
-          format
-          slug
-          documents {
-            edges {
-              node {
-                ...on Document {
-                  _sys {
-                    extension
-                    path
-                    relativePath
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
-      `
-  const result = await resolve({
-    database,
-    query,
-    variables: {},
-  })
+  const { collection, database, useDefaultValues, documents } = args
   let error = false
-  const documents: any[] = result.data.collection.documents.edges
   for (let i = 0; i < documents.length; i++) {
     const node = documents[i].node
-    const fullPath = p.join(rootPath, node._sys.path)
-    logger.info(`Checking document: ${fullPath}`)
+    const relativePath = node.path.replace(`${collection.path}/`, '')
+    logger.info(`Checking document: ${node.path}`)
```

Comment:
```suggestion
```
Now that we don't exit after a failure, I think it's more readable if we only output the issues

Reply:
yeah, much cleaner.